casper/src/test/scala/coop/rchain/casper/rholang/RuntimeManagerTest.scala

```diff
@@ -439,7 +455,7 @@ class RuntimeManagerTest extends AnyFlatSpec with Matchers {
         deploy1 <- ConstructDeploy.sourceDeployNowF(
                     s"""new return in { for(nn <- @"nn"){ nn!("value", *return) } } """
                   )
-        result1 <- mgr.captureResults(hash, deploy1)
+        result1 <- mgr.spawnRuntime >>= { _.captureResults(hash, deploy1) }
```

Comment:
Why is now a new runtime instance needed?

Reply:
If you look at most the methods in runtimeManager, it `new` in most method. Here is to remove only test used method `captureResults` in `RuntimeManager` trait